LayoutTests/http/tests/lockdown-mode/heic-shown-in-lockdown-mode.http-expected.txt

```diff
@@ -0,0 +1,11 @@
+Tests the behavior of Accept Headers in Lockdown Mode. We do not restrict our Accept Headers to only the allowed formats when requested by an insecure source.
+
+On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".
+
+
+PASS successfullyParsed is true
+
+TEST COMPLETE
```

Comment:
This test is likely not written correctly as `TEST COMPLETE` is printed *before* the test lines.

Reply:
Usually, such tests end up being flaky on the bots.